crates/release_plz_core/src/next_ver.rs

```diff
@@ -425,14 +425,32 @@ impl Updater<'_> {
         let mut packages_to_check_for_deps: Vec<&Package> = vec![];
         let mut packages_to_update = PackagesUpdate { updates: vec![] };
         for p in &self.project.packages {
-            let semver_check = self.req.get_package_config(&p.name).semver_check();
-            let diff = get_diff(
+            let run_semver_check = self.req.get_package_config(&p.name).semver_check();
+            let mut diff = get_diff(
                 p,
-                semver_check,
+                run_semver_check,
                 registry_packages,
                 repository,
                 &self.project.root,
             )?;
+
+            let package_path = {
+                let relative_path = p
+                    .package_path()?
+                    .strip_prefix(&self.project.root)
+                    .context("error while retrieving package_path")?;
+                repository.directory().join(relative_path)
+            };
```

Comment:
This looks great, thanks!
Just one comment: can you extract this into a function? It's also repeated at line 610.

Reply:
fix findings. Can you pls review again?